src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #F3F3F3;
+  border-radius: 5px;
+  font-family: 'Roboto', sans-serif;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    padding: 32px 19px 0;
```

Comment:
just set paddings for the entire card to avoid setting it to each element inside